astrapy/defaults.py

```diff
@@ -1,10 +1,12 @@
 DEFAULT_AUTH_PATH = "/api/rest/v1/auth"
-DEFAULT_BASE_PATH = "/api/json/v1"
+
+DEFAULT_JSON_API_PATH = "/api/json"
+DEFAULT_JSON_API_VERSION = "v1"
```

Comment:
Yeah, perhaps adding a unit test that provides the same api_path and api_versions with of without trailing _and leading_ slash, and then checks you always get the same self.api_path and self.base_path , would be a good sanity check.